docs/topics/migrations.txt

```diff
@@ -276,6 +276,37 @@ class to make it importable::
 Please refer to the notes about :ref:`historical-models` in migrations to see
 the implications that come along.
 
+.. _initial-property:
```

Comment:
Please change `initial-property` to `migration-initial-property` (slugify the section title)